tests/ashley/draftjs_exporter/test_draftjs_exporter_decorator_latex.py

```diff
@@ -127,3 +129,30 @@ def test_custom_decorator_displaytex_no_malformed(self):
             ),
             '<span class="ashley-latex-display"></span>',
         )
+
+    @mock.patch("ashley.editor.decorators.render_children")
```

Comment:
It is better to mock `draftjs_exporter.defaults.render_children` WDYT ?

Reply:
I just want to make sure it gets called, so to me, it's the straightest way and it makes sure it gets called from this custom file.
On top of that if i patch the original i have to declare a more confusing props that has an attribute children that calls render_children , I thought it could be confusing it reading it, I can update it though